Scripts/configure-ODBC-DSN.ps1

```diff
@@ -0,0 +1,35 @@
+﻿param (
+    [parameter(Mandatory=$true)]
+    [string] $lansaVersion
+)
+
+$DSNNames = (Get-OdbcDsn).Name
+
+# AZURESQL
+
+Write-Host("Updating Server name and database name for AZURESQL")
+
+if($DSNNames -contains "AZURESQL" -and (Get-OdbcDsn -Name "AZURESQL").Platform -eq "32-bit"){
+    $sqlserver = "db-regression-" + $lansaVersion + ".database.windows.net" 
+    Set-OdbcDsn -Name "AZURESQL" -DsnType "System" -Platform "32-bit" -SetPropertyValue @("Server=$sqlserver", "Database=$lansaVersion")
+}
+
+# Oracle
+
+Write-Host("Updating Server name for Oracle")
+
+if($DSNNames -contains "ora19cdb" -and (Get-OdbcDsn -Name "ora19cdb").Platform -eq "32-bit"){
```

Comment:
The DSN Name is ORACLE. You are using an old AMI.

Reply:
Done.